keepsorted/options.go

```diff
@@ -18,12 +18,12 @@ import (
 	"fmt"
 	"reflect"
 	"regexp"
-	"strconv"
 	"strings"
 	"unicode"
 
 	"go.uber.org/multierr"
 	"golang.org/x/exp/slices"
+	"math/big"
```

Comment:
DO NOT SUBMIT. Still trying to figure out why intellij doesn't like formatting code for me...